operations/network/network_management_install_guide/aruba/canu_installation.md

```diff
@@ -0,0 +1,21 @@
+# Canu
+
+## Installation
+
+### Prerequisites
+
+In order to run CANU, both python3 and pip3 need to be installed.
+
+### Installation
+
```

Comment:
Might want to show how to clone the git repo here.

Reply:
I had the discussion with Sean about this. Not sure if we can do it for initial release as we would need to still build out the cloning. Thoughts?